### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/guide/playground.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#523] [Announcements]
# https://plugin-md-enhance.vuejs.press/guide/playground.html

The plugin provides you playground support. Config tab TS tab JS Usage You should add presets through playground.presets in plugin options. To use playground, you should use a c...

https://plugin-md-enhance.vuejs.press/guide/playground.html

**Comment by YY88Xu** [comment]
引入 ant-design-vue 的文件 https://unpkg.com/browse/ant-design-vue@2.1.6/lib/index.js 报错，Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.

**Reply by Mister-Hope** [reply]
会用cdn不？不会看首页。

![Screenshot_2023-04-16-09-29-02-34_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://user-images.githubusercontent.com/33315834/232261253-aa39e3bb-b113-4187-b090-da0b5415a27a.jpg)

**Comment by YY88Xu** [comment]
js 类型文件引入报错，怎么修改
